style.css

```diff
@@ -1,13 +1,926 @@
+  align-items: center;
+  background-image: url(./Images/background\ Capstone.jpg);
+  background-color: rgba(245, 245, 245, 0.17);
+}
+
+.main-program h2 {
+  color: #fff;
+  padding-top: 30px;
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.icon-container {
+  padding: 0;
+  margin: 0;
+}
+
+.program-icon {
+  width: 45px;
+  margin: 0;
+  padding: 0;
+}
+
+.activity {
+  display: flex;
+  margin: 0 3% 5px 3%;
+  align-items: center;
+  background-color: rgba(245, 245, 245, 0.15);
+}
+
+.activity div {
+  width: 45%;
+  padding: 1.5rem 0 1.5rem 0.5rem;
+  margin-right: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.activity span {
+  color: #ec5242;
+  font-size: 18px;
+}
+
+.activity p {
+  color: #fff;
+  font-size: 0.8rem;
+  width: 45%;
+}
+
+.join-global-forum {
+  all: unset;
+  color: #fff;
+  background-color: #ec5242;
+  outline: none;
+  padding: 2rem;
+  margin-top: 2.5rem;
+  margin-bottom: 1.8rem;
+}
+
+/* ----------------- SPEAKERS ------------------- */
+
+.section-speakers {
+  color: #272a31;
+  padding-top: 30px;
+  margin-bottom: 3rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.section-speakers h2 {
+  font-size: 20px;
+  font-weight: 400;
+}
+
+#speaker-grid-container {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.speaker-full-container {
+  display: flex;
+  margin: 0 2.5% 5px 2.5%;
+  margin-bottom: 20px;
+}
+
+.speaker-picture-container {
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.race-flag {
+  width: 65px;
+  position: absolute;
+  float: left;
+  z-index: 1;
+}
+
+.speaker-picture {
+  width: 65%;
+  height: 75%;
+  margin-left: 8%;
+  position: relative;
+  margin-top: 8%;
+  z-index: 10;
+}
+
+.speaker-profile-data {
+  width: 65%;
+  margin-left: 12px;
+}
+
+.speaker-profile-data-name {
+  color: #272a31;
+  font-size: 1rem;
+  font-weight: 700;
+  padding-bottom: 10px;
+}
+
+.speaker-profile-data-role {
+  color: #ec5242;
+  font-size: 0.7rem;
+  font-weight: 700;
+  margin-bottom: 0.6rem;
+  font-style: italic;
+}
+
+.speaker-profile-data-description {
+  color: #272a31;
+  font-size: 0.55rem;
+  font-weight: 700;
+  padding-bottom: 5px;
+}
+
+.down-arrow-icon {
+  width: 1rem;
+  margin-left: 12px;
+}
+
+/* ||||||| ----------------- ABOUT MOBILE SECTION ------------------- |||||||| */
+
+.about-section {
+  background-color: #f8f6f6;
+}
+
+.about-intro-container {
+  margin: 0 2.5% 0 2.5%;
+  display: flex;
+  flex-direction: column;
+  padding-top: 12px;
+  padding-bottom: 55px;
+  color: #ec5242;
+  justify-content: center;
+}
+
+.about-intro-container h2 {
+  font-size: 1.4rem;
+  font-weight: 500;
+  margin-bottom: 18px;
+  align-self: center;
+  margin-top: 78px;
+  text-align: center;
+}
+
+.about-intro-title-conatiner h1 {
+  font-size: 2.3rem;
+  font-weight: 900;
+  text-align: center;
+}
+
+.about-description-container {
+  background-color: #fff;
+}
```

Comment:
- [ ] Please make sure to add a bit of margin on the left and right side and align the text to center for mobile version 📌

Your design | Given design
---| ---|
![image](https://github.com/HFG43/Capstone1/assets/106119105/55785004-afd9-401b-9e95-1203b8d7adb5) | ![image](https://github.com/HFG43/Capstone1/assets/106119105/921de0c5-da76-4f3d-91a0-0b5a790f9a3a)